nbs/core.ipynb

```diff
@@ -25,9 +25,8 @@
    "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #17.                Y_h: pd.DataFrame, # Base forecasts with columns <code>ds</code> and models to reconcile indexed by <code>unique_id</code>.
</pre><p>Since we are changing the way the documentation is generated, let's remove the old documentation (the # and descriptions after each argument). </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/hierarchicalforecast/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>